### DISCUSSION TITLE
bash: ./linux-wallpaperengine: Is a directory

### DISCUSSION THREAD

**Discussion by vitalii-py** [Almamu/linux-wallpaperengine#142] [Q&A]
Hi, successfully built the program for fedora 37, but after running ./linux-wallpaperengine is shown as a directory and I can't install the wallpaper.

**Comment by Almamu** [comment]
Can you do an ```ls -lah``` on the build directory?

**Reply by vitalii-py** [reply]
Did it.
Now I get how it works. I expected that I could run the app from everywhere after I created it.
Thank you very much.

**Reply by Almamu** [reply]
There's no install step yet for the app, so you'll have to add it to PATH manually or keep it in a folder like /opt/linux-wallpaperengine/ for easy invoking.

In the future I plan on adding install steps, but I don't think it is worth it yet...
